Dockerfile-devel-curve

```diff
@@ -41,22 +41,42 @@ RUN yum install -y \
     centos-release-scl-rh \
     which && \
     if [ "$TARGETARCH" = "amd64" ]; then \
-    OCI_ARCH=x86_64; \
+    BAZEL_ARCH=x86_64; \
     elif [ "$TARGETARCH" = "arm64" ]; then \
-    OCI_ARCH=aarch64; \
+    BAZEL_ARCH=arm64; \
     else \
-    OCI_ARCH=x86_64; \
+    BAZEL_ARCH=x86_64; \
     fi && \
-    wget https://curve-build.nos-eastchina1.126.net/bazelbuild/bazelisk/bazelisk-linux-${OCI_ARCH} && \
-    mv bazelisk-linux-${OCI_ARCH} /usr/bin/bazelisk && \
+    wget https://curve-build.nos-eastchina1.126.net/bazelbuild/bazelisk/bazelisk-linux-${BAZEL_ARCH} && \
+    mv bazelisk-linux-${BAZEL_ARCH} /usr/bin/bazelisk && \
     ln -s /usr/bin/bazelisk /usr/bin/bazel && \
-    wget https://curve-build.nos-eastchina1.126.net/bazelbuild/4.2.2/bazel-4.2.2-linux-${OCI_ARCH} && \
-    mkdir -p /root/.cache/bazelisk/downloads/bazelbuild/bazel-4.2.2-linux-${OCI_ARCH}/bin && \
-    mv bazel-4.2.2-linux-${OCI_ARCH} /root/.cache/bazelisk/downloads/bazelbuild/bazel-4.2.2-linux-${OCI_ARCH}/bin/bazel && \
-    wget https://curve-build.nos-eastchina1.126.net/bazelbuild/0.17.2/bazel-0.17.2-linux-${OCI_ARCH} && \
-    mkdir -p /root/.cache/bazelisk/downloads/bazelbuild/bazel-0.17.2-linux-${OCI_ARCH}/bin && \
-    mv bazel-0.17.2-linux-${OCI_ARCH} /root/.cache/bazelisk/downloads/bazelbuild/bazel-0.17.2-linux-${OCI_ARCH}/bin/bazel && \
-    chmod +x /root/.cache/bazelisk/downloads/bazelbuild/bazel-4.2.2-linux-${OCI_ARCH}/bin/bazel /root/.cache/bazelisk/downloads/bazelbuild/bazel-0.17.2-linux-${OCI_ARCH}/bin/bazel /usr/bin/bazel /usr/bin/bazelisk && \
+    wget https://curve-build.nos-eastchina1.126.net/bazelbuild/4.2.2/bazel-4.2.2-linux-${BAZEL_ARCH} && \
+    mkdir -p /root/.cache/bazelisk/downloads/bazelbuild/bazel-4.2.2-linux-${BAZEL_ARCH}/bin && \
+    mv bazel-4.2.2-linux-${BAZEL_ARCH} /root/.cache/bazelisk/downloads/bazelbuild/bazel-4.2.2-linux-${BAZEL_ARCH}/bin/bazel && \
+    wget https://curve-build.nos-eastchina1.126.net/bazelbuild/0.17.2/bazel-0.17.2-linux-${BAZEL_ARCH} && \
+    mkdir -p /root/.cache/bazelisk/downloads/bazelbuild/bazel-0.17.2-linux-${BAZEL_ARCH}/bin && \
+    mv bazel-0.17.2-linux-${BAZEL_ARCH} /root/.cache/bazelisk/downloads/bazelbuild/bazel-0.17.2-linux-${BAZEL_ARCH}/bin/bazel && \
+    chmod +x /root/.cache/bazelisk/downloads/bazelbuild/bazel-4.2.2-linux-${BAZEL_ARCH}/bin/bazel /root/.cache/bazelisk/downloads/bazelbuild/bazel-0.17.2-linux-${BAZEL_ARCH}/bin/bazel /usr/bin/bazel /usr/bin/bazelisk && \
+    if [ "$TARGETARCH" = "arm64" ]; then \
+    yum install -y epel-release && \
+    yum install -y \
+    devtoolset-9-gcc \
```

Comment:
1. We'd better have indentation inside `if` block, or else it will be hard to tell the difference
2. If there is some common part between X86 and ARM, this part should be extracted outside of `if`

Reply:
> 1. We'd better have indentation inside `if` block, or else it will be hard to read
> 2. If there is some common part between X86 and ARM, this part should be extracted outside of `if`

fix